molgenis-core-ui/README.md

```diff
@@ -0,0 +1,83 @@
+# MOLGENIS core UI
+
+## Javascript
+We use the [frontend maven plugin](https://github.com/eirslett/frontend-maven-plugin)
+to allow maven to install node and run npm and other javascript tools.
+
+The javascript files in and other static resources src/main/javascript are packed into bundles using webpack.
+
+### Directory structure
+```
+|-- molgenis-core-ui
+    |-- README.md // this file
+    |-- gulpfile.js // javascript build actions, used to write version from pom.xml to package.json
+    |-- package.json // describes what npm packages are used
+    |-- pom.xml // the maven buildfile
+    |-- webpack.config.js // webpack configuration, lists the entry points for the bundles and the loaders
+    |-- webpack.test.config.js // webpack configuration for tape tests
+    |-- node // node gets installed here by frontend maven plugin
+    |   |-- node
+    |   |-- npm
+    |   |-- npm.cmd
+    |-- node_modules // npm modules listed in package.json get downloaded to here by npm and read by webpack and node
+    |   |-- [...]
+    |-- src
+    |   |-- main
+    |   |   |-- [...]
+    |   |   |-- javascript
+    |   |   |   |-- molgenis-global-ui-webpack.js // bundle files specify the contents of the bundles
+    |   |   |   |-- molgenis-global-webpack.js
+    |   |   |   |-- molgenis-vendor-webpack.js
```

Comment:
ouch, what a huge number of webpack configs.
Is this only once or would each module need to this as well?